packages/remix/src/client/uiComponents.tsx

```diff
@@ -20,12 +22,14 @@ export const SignIn = (props: SignInProps) => {
 
 export const SignUp = (props: SignUpProps) => {
   const { signUpUrl } = useClerkRemixOptions();
+  const { routing: _routing, path: _path, ...restProps } = props;
+
   if (signUpUrl) {
     return (
       <BaseSignUp
         routing='path'
         path={signUpUrl}
-        {...props}
+        {...restProps}
```

Comment:
❓ Why the current implementation overriding the `routing = 'path'` if `signUpUrl` was used and `routing` prop was passed to `SignUp` component?
Could we add a test for that we haven't already?

Reply:
Yeah you are right the current implementation is overriding `routing` or `path` if the props are passed to the `SignUp` component, I've deployed a fix for that